src/windows/pdb.rs

```diff
@@ -423,13 +424,14 @@ impl<'s> PDBData<'s> {
                         len: procedure.len,
                     },
                     lines,
+                    module_index,
                 );
             }
             SymbolData::Block(block) => {
-                self.add_block(module_info, block, collector, lines)?;
+                self.add_block(module_index, module_info, block, collector, lines)?;
```

Comment:
I was going for a different kind of consistency: I added `module_index` just before `module_info` everywhere, because I consider `(module_index, module_info)` to be something that appears in pairs. So I think I'll make `add_block` consistent with `add_sepcode` by moving `module_index` to be the first parameter to `add_sepcode`.